lms/djangoapps/learner_recommendations/serializers.py

```diff
@@ -39,6 +39,24 @@ def get_prospectusPath(self, instance):
         return f"course/{url_slug}"
 
 
+class CrossProductCourseSerializer(serializers.Serializer):
+    """Serializer for a cross product recommended course"""
+    key = serializers.CharField()
+    uuid = serializers.UUIDField()
+    title = serializers.CharField()
+    image = CourseImageSerializer()
+    prospectusPath = serializers.SerializerMethodField()
+    owners = serializers.ListField(
+        child=CourseOwnersSerializer(), allow_empty=True
+    )
+    activeCourseRun = ActiveCourseRunSerializer(source="active_course_run")
+    courseType = serializers.CharField(source="course_type")
+
+    def get_prospectusPath(self, instance):
+        url_slug = instance.get("url_slug")
+        return f"course/{url_slug}"
+
+
```

Comment:
Since this serializer has fields almost exactly similar to the existing `RecommendedCourseSerializer` except for the extra field `courseType`, It was previously declared to extend from the `RecommendedCourseSerializer` and just declared with that extra field. The reason it has now been declared as it's own serializer not extending from the exiting serializer is that if the existing serializer were to change, it would affect our `CrossProductCourseSerializer`, so this abstracts this serializer and removes its dependancy.